.zuul.yaml

```diff
@@ -75,6 +75,12 @@
     cloud-provider-openstack-acceptance-test-flexvolume-cinder:
       jobs:
         - cloud-provider-openstack-acceptance-test-flexvolume-cinder:
+            files:
+              - ^cmd/cinder-flex-volume-driver/.*
+              - ^pkg/flexvolume/.*
```

Comment:
pkg/cloudprovider/ is used as well

Reply:
Yeah, just saw that.

BTW, do you know if there is anyone using the cinder flex driver? As CSI support has been GA since v1.13, do you think it's necessary for us to maintain this driver?

Reply:
maybe you want to ask in some mail list about this ? not sure if someone is running this in their production/test env..

Reply:
Disabling CI job doesn't mean we will remove the code in repo nor images on docker hub, it means we don't have any contributor interested in maintaining the driver.

Reply:
I am ok to do so,  as this will prevent quality control of the spcified code, e.g will someone get some image from docker hub that doesn't have the CI run for the driver so they don't know it's been tested and start to use it?
if that's not gonna happen, then we can disable CI ..

Reply:
1. The latest image tag is not supposed to be production-ready.
2. We won't do release for Cinder FlexVolume Plugin from the next release (I am the person to do the release, so it's on my release note now).

Reply:
> I am ok to do so, as this will prevent quality control of the spcified code, e.g will someone get some image from docker hub that doesn't have the CI run for the driver so they don't know it's been tested and start to use it?
> if that's not gonna happen, then we can disable CI ..

However, I would be very happy if you are interested in maintaining this plugin and wanna be the owner

Reply:
no, I have other items so can't do the job :)
the reason I am having this kind of question is to avoid break anyone who is using this
so as you mentioned, we will never have further release, I agree we need stop CI

thanks